app/utility/op_control.py

```diff
@@ -35,6 +35,11 @@ def __init__(self, dao):
             return True
         return False
 
+    async def init_operation(self, operation):
+        state = await self.get_state(operation)
+        if state != 'PAUSED' and state != 'CANCELED':
```

Comment:
instead of 'paused' and 'cancelled' can we use the 'cancel and pause' variables in the enum (in op_control)? so it would be OpState.CANCEL. this would be cleaner than a raw string here.

Reply:
Sure, that's doable/done.